src/ctia/stores/es/store.clj

```diff
@@ -72,8 +72,8 @@
 
 (defrecord RelationshipStore [state]
   IRelationshipStore
-  (create-relationship [_ new-relationship]
-    (rel/handle-create-relationship state new-relationship))
+  (create-relationship [_ new-relationships]
+    (rel/handle-create-relationship state new-relationships))
```

Comment:
all of these, since they are namespaced already could just be handle-create and handle-read ...